doc/README.md

```diff
@@ -36,6 +36,9 @@ Check out our test results:
 
 ## MongooseIM platform components
 
+<img src="MongooseIM Platform components.png" alt="MongooseIM platform schema" />
```

Comment:
When I read the diagram from top to bottom, the natural "thinking" for me is that the "Android Mangosta" block is built from "iOS XMPPFramework" and "Android Smack". The very similar flow I have for the other block "iOS Mangosta". Maybe it'd be more natural if you put blocks related to Android only under "Android Mangosta" and the same for "iOS Mangosta".

Also I can see that block representing XMPP and XMPP libraries have the same background. Probably it'b good to have the same background for all REST-related blocks, now they have a bit different background.